src/ontology/mondo-edit.obo

```diff
@@ -185681,7 +185682,6 @@ subset: orphanet_rare {source="Orphanet:90"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "Arg1 deficiency" RELATED []
-synonym: "arginase deficiency" EXACT [DOID:9278, icd11.foundation:1619102598, OMIM:207800, Orphanet:90]
```

Comment:
You can keep this synonym, even if it is redundant with the term label. This way, we keep provenance (x-ref) of where this name is used.